src/distribution_spatial.cpp

```diff
@@ -306,25 +306,25 @@ Position MeshSpatial::sample(uint64_t* seed) const
 
 PointCloud::PointCloud(pugi::xml_node node)
 {
-  std::vector<double> coords;
-
   if (check_for_node(node, "coords")) {
     point_cloud_ = get_node_position_array(node, "coords");
   } else {
     fatal_error("No coordinates were provided for the PointCloud "
                 "spatial distribution");
   }
 
-  std::vector<double> strengths(point_cloud_.size(), 1.0);
+  std::vector<double> strengths;
```

Comment:
Is this a style question?  FWIW, this was modeled on `MeshSpatial` that initializes this to 1.0 and then only changes if the XML data exists.  If this change happens, we may want to change `MeshSpatial` accordingly.

Reply:
I moved this below for size considerations. In the case that there's a "strengths" entry on the XML node we'd be storing two vectors of size `point_cloud_`. I'm happy to make an issue and update `MeshSpatial` as well in a separate PR!